pkg/runctl/run_manager.go

```diff
@@ -234,6 +234,11 @@ func (c *runManager) copySecretsToRunNamespace(ctx *runContext) (string, []strin
 		return "", nil, errors.Wrap(err, "failed to copy pipeline clone secret")
 	}
 
+	_, err = c.copyElasticSearchCredentialToRunNamespace(ctx, secretHelper)
```

Comment:
```suggestion
	ctx.elasticSearchCredentilaName, err = c.copyElasticSearchCredentialToRunNamespace(ctx, secretHelper)
```
Extend the context to be able to hold the elasticSearchCredentilaName